api/types/trust.go

```diff
@@ -38,6 +38,10 @@ const (
 	// certificates but rather is an authority that signs tokens, however it behaves
 	// much like a CA in terms of rotation and storage.
 	JWTSigner CertAuthType = "jwt"
+	// CertAuthTypeAll is a special type that represents all CertAuthTypes.
+	// DEPRECATED, DELETE IN 14.0.0. For more information see:
```

Comment:
I talked this over with @rosstimothy and he suggested giving folks a couple of releases which seems reasonable as a user. I like dropping it in v13 just because that's right around the corner and the work is top of mind.